test/test_twodnavierstokes.jl

```diff
@@ -56,30 +56,39 @@ function test_twodnavierstokes_stochasticforcingbudgets(dev::Device=CPU(); n=256
 
   TwoDNavierStokes.set_zeta!(prob, 0*x)
   E = Diagnostic(TwoDNavierStokes.energy,      prob, nsteps=nt)
-  D = Diagnostic(TwoDNavierStokes.dissipation, prob, nsteps=nt)
-  R = Diagnostic(TwoDNavierStokes.drag,        prob, nsteps=nt)
-  W = Diagnostic(TwoDNavierStokes.work,        prob, nsteps=nt)
-  diags = [E, D, W, R]
+  D = Diagnostic(TwoDNavierStokes.energy_dissipation, prob, nsteps=nt)
+  R = Diagnostic(TwoDNavierStokes.energy_drag,        prob, nsteps=nt)
+  W = Diagnostic(TwoDNavierStokes.energy_work,        prob, nsteps=nt)
+  Z = Diagnostic(TwoDNavierStokes.enstrophy,      prob, nsteps=nt)
+  DZ = Diagnostic(TwoDNavierStokes.enstrophy_dissipation, prob, nsteps=nt)
+  RZ = Diagnostic(TwoDNavierStokes.enstrophy_drag,        prob, nsteps=nt)
+  WZ = Diagnostic(TwoDNavierStokes.enstrophy_work,        prob, nsteps=nt)
+  diags = [E, D, W, R, Z, DZ, WZ, RZ]
 
   stepforward!(prob, diags, nt)
   TwoDNavierStokes.updatevars!(prob)
 
-  E, D, W, R = diags
+  E, D, W, R, Z, DZ, WZ, RZ = diags
   t = round(μ*cl.t, digits=2)
 
   i₀ = 1
   dEdt = (E[(i₀+1):E.i] - E[i₀:E.i-1])/cl.dt
-  ii  = (i₀):E.i-1
+  dZdt = (Z[(i₀+1):Z.i] - Z[i₀:Z.i-1])/cl.dt
+  ii = (i₀):E.i-1
   ii2 = (i₀+1):E.i
 
   # dEdt = W - D - R?
   # If the Ito interpretation was used for the work
   # then we need to add the drift term
   # total = W[ii2]+ε - D[ii] - R[ii]      # Ito
   total = W[ii2] - D[ii] - R[ii]        # Stratonovich
+  totalZ = WZ[ii2] - DZ[ii] - RZ[ii]
 
   residual = dEdt - total
   isapprox(mean(abs.(residual)), 0, atol=1e-4)
+
```

Comment:
This needs to be changed to:

```julia
 residual = dEdt - total
residualZ = dZdt - totalZ

return isapprox(mean(abs.(residual)), 0, atol=1e-4) && isapprox(mean(abs.(residualZ)), 0, atol=(kf^2)*1e-4)
```
otherwise only the answer regarding the enstrophy budget is taken into consideration.